File: app/src/main/java/com/github/premnirmal/ticker/network/SuggestionApi.java
package com.github.premnirmal.ticker.network;

import java.util.List;

import retrofit.http.GET;
import rx.Observable;

/**
 * Created by premnirmal on 12/21/14.
 */
public interface SuggestionApi {


    @GET("/Lookup/json")
    Observable<List<Suggestion>> getSuggestions(@retrofit.http.Query("input") String query);

}

File: app/src/main/kotlin/com/github/premnirmal/ticker/network/SuggestionApi.kt
package com.github.premnirmal.ticker.network

import com.github.premnirmal.ticker.network.data.SuggestionsNet
import retrofit2.http.GET
import retrofit2.http.Query

/**
 * Created by premnirmal on 3/3/16.
 */
interface SuggestionApi {

  @GET("search?quotesCount=20&newsCount=0&listsCount=0&enableFuzzyQuery=false")
  suspend fun getSuggestions(@Query("q") query: String): SuggestionsNet

}
